README.md

```diff
@@ -24,21 +26,28 @@ Create new project on Shopsys Framework easily from this package following one o
 ## Documentation
 For documentation of Shopsys Framework itself see [Shopsys Framework Knowledge Base](https://github.com/shopsys/shopsys/blob/master/docs/index.md).
 
-Documentation of the specific project built on Shopsys Framework should be in [Project Documentation](https://github.com/shopsys/project-base/blob/master/docs/index.md).
+Documentation of this specific Demoshop project should be in [Project Documentation](https://github.com/shopsys/demoshop/blob/master/docs/index.md).
 
 ## Contributing
-Thank you for your contributions to Shopsys Project-base.
-Together we are making Shopsys Framework better.
+If you have some ideas or you want to help to improve Shopsys Framework Demoshop, let us know!
+We are looking forward to your insights, feedback, and improvements.
+Thank you for helping us making Shopsys Framework Demoshop better.
 
-This repository is READ-ONLY.
-If you want to [report issues](https://github.com/shopsys/shopsys/issues/new) and/or send [pull requests](https://github.com/shopsys/shopsys/compare),
-please use the main [Shopsys repository](https://github.com/shopsys/shopsys).
-
-Please, check our [Contribution Guide](https://github.com/shopsys/shopsys/blob/master/CONTRIBUTING.md) before contributing.
+You can find all the necessary information in our [Contribution Guide](./CONTRIBUTING.md). 
 
 ## Support
-What to do when you are in troubles or need some help? Best way is to contact us on our Slack [http://slack.shopsys-framework.com/](http://slack.shopsys-framework.com/)
+What to do when you are in troubles or need some help?
+The best way is to contact us on our [Slack](http://slack.shopsys-framework.com/).
+
+If you are experiencing problems during installation or running Shopsys Framework on Docker,
+please see our [Docker troubleshooting](./docs/docker/docker-troubleshooting.md).
```

Comment:
Docker troubleshooting does not cover problems with installation... I dont think that you should change something, just sayin that if someone uses native installation that Docker troubleshooting wont help you :)